jmeter-java-dsl-elasticsearch-listener/pom.xml

```diff
@@ -77,6 +77,12 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
This dependency is already added at the beginning of dependencies section. Any reason to re ad it with a fixed version?